server/telemetry/telemetry.go

```diff
@@ -0,0 +1,62 @@
+package telemetry
+
+import (
+	mmtelemetry "github.com/mattermost/mattermost-plugin-api/experimental/telemetry"
+)
+
+type Telemetry struct {
+	tracker mmtelemetry.Tracker
+}
+
+func NewTelemetry(tracker mmtelemetry.Tracker) *Telemetry {
+	return &Telemetry{
+		tracker: tracker,
+	}
+}
+
+func (t *Telemetry) UpdateTracker(tracker mmtelemetry.Tracker) {
+	t.tracker = tracker
+}
+
+func (t *Telemetry) TrackInstall(appID string, appType string) {
```

Comment:
Would the signature be simpler with 
```suggestion
func (t *Telemetry) TrackInstall(appID, appType string) {
```
?